release.json

```diff
@@ -1,17 +1,17 @@
 {
-    "base_branch": "main",
+    "base_branch": "7.61.x",
     "current_milestone": "7.61.0",
     "last_stable": {
         "6": "6.53.0",
         "7": "7.59.0"
     },
     "nightly": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "0059a287d5543305c01a098740ba328aef7fa8ff",
-        "OMNIBUS_RUBY_VERSION": "d365e483ee05a13e55eeb5208d11452c5b65afbb",
+        "INTEGRATIONS_CORE_VERSION": "7.61.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.61.x",
+        "OMNIBUS_RUBY_VERSION": "7.61.x",
```

Comment:
Please stick to pinned commits for omnibus repos in the nightly config, the commit is used in computing the git cache key, and using a branch would cause us to potentially use stalled cache

Reply:
Gotcha. I'll manually update that part. We'll need to modify the script as well.

Reply:
Updated (that's the same commit)